config/environments/staging.rb

```diff
@@ -1 +1 @@
-load(File.expand_path("../production.rb"))
+load(File.join(File.dirname(__FILE__), 'production.rb'))
```

Comment:
FYI I usually use this since I think it reads nicer / works nicely with deeply nested traversal:
`File.expand_path('../production.rb', __FILE__)`